pages/windows/msg.md

```diff
@@ -0,0 +1,23 @@
+# msg
+
+> Send a message to a specific user or session.
+
+- Send a message to a specified user or session:
+
+`msg {{username|session_name|session_id}} {{message}}`
+
+- Send a message from stdin:
```

Comment:
In this case, which user|session does the message go to ?

Reply:
Good point, it needs to be specified. 🤦‍♂️